test/nodes/classical-register_spec.js

```diff
@@ -24,7 +24,8 @@ describe('ClassicalRegisterNode', function() {
   });
 
   it('execute quantum circuit initilization command if it is the last register node connected', function(done) {
```

Comment:
```suggestion
  it('execute quantum circuit command if last register is connected', function(done) {
```
I think this test name is a bit verbose and can be shortened.